routemodel/data_retrieval/get_weather.py

```diff
@@ -3,58 +3,82 @@
 sys.path.append(os.path.dirname(__file__))
 import requests
 import csv
-import json
 
 from config import WEATHER_API_KEY
+
 ONE_CALL_BASE = 'https://api.openweathermap.org/data/2.5/onecall?'
-PATH = os.path.join(os.path.dirname(__file__), '..', 'routes\ASC2021\ASC2021_draft.csv')
+PATH = os.path.join(os.path.dirname(__file__), '..', 'routes/ASC2021/ASC2021_draft.csv')
+
+
+def get_weather(lat, long, weather_api_result):
+    """
+    Calls OpenWeather API to get wind data and returns data in a list
+    @param lat: string containing latitude value
+    @param long: string containing longitude value
+    @param weather_api_result: list of lists containing weather data
+    @return: list of lists containing weather data
+
+    """
+    query_url = ONE_CALL_BASE + 'lat=' + lat + '&lon=' + long + '&exclude=hour,daily&units=metric&appid=' + WEATHER_API_KEY
```

Comment:
kinda miss the `.format` string you had in your version?

Reply:
aha sorry yes I wasn't sure how much of the get_weather file I was "allowed" to change. Will change this